erp/views.py

```diff
@@ -234,6 +235,19 @@ def _get_search_qs_and_context(commune_slug, lat, lon, what, where):
     }
 
 
+def _translate_erp(erp: Erp, target_lang: str):
```

Comment:
I think could have been a method of the erp object no ? This could ease the use of translated fields elsewhere (exports, api, etc).